bazel/envoy_mobile_repositories.bzl

```diff
@@ -15,24 +15,6 @@ def envoy_mobile_repositories():
     python_repos()
 
 def upstream_envoy_overrides():
```

Comment:
Can this function be removed altogether if it's empty?

Reply:
not empty. There is more stuff below

Reply:
ah, I see, the GitHub UI threw me.